### DISCUSSION TITLE
Add an extra feature to the dashboard native filter for time grain

### DISCUSSION THREAD

**Discussion by AHZenan** [apache/superset#20191] [Ideas]
Hi 

Would it be possible to add an extra feature on the time grain filter whereby we can filter the values within the time grain. The feature already exists for the other filters . See example below :
![Screenshot 2022-05-25 171648](https://user-images.githubusercontent.com/81419019/170310266-53509ed3-b718-4ef9-a503-024232da4b23.jpg)

**Comment by sfirke** [comment]
Agreed this would be helpful.  I have a time-series chart where the data reports at 5-minute intervals (a rain gauge) so I'd like to exclude filter options to show by the second and minute.  Should this be an issue, rather than an idea/discussion?

**Comment by sfirke** [comment]
Realizing this is a duplicate of https://github.com/apache/superset/discussions/20191.

**Comment by jakejh** [comment]
Yes, please please allow selection of which granularities to show in a time grain filter! If I have a chart where things are happening on the day, week, or month timescale, I don't need to see second, minute, 5 minute, 10 minute, etc.
